src/test/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/helper/EntityDataLoader.kt

```diff
@@ -0,0 +1,158 @@
+package uk.gov.justice.digital.hmpps.nomisuserrolesapi.helper
+
+import org.springframework.data.repository.findByIdOrNull
+import uk.gov.justice.digital.hmpps.nomisuserrolesapi.jpa.LAAAdminUser
+import uk.gov.justice.digital.hmpps.nomisuserrolesapi.jpa.LAAAdminUserPk
+import uk.gov.justice.digital.hmpps.nomisuserrolesapi.jpa.LAAGeneralUser
+import uk.gov.justice.digital.hmpps.nomisuserrolesapi.jpa.LAAGeneralUserPk
+import uk.gov.justice.digital.hmpps.nomisuserrolesapi.jpa.Staff
+import uk.gov.justice.digital.hmpps.nomisuserrolesapi.jpa.UserPersonDetail
+import uk.gov.justice.digital.hmpps.nomisuserrolesapi.jpa.repository.CaseloadRepository
+import uk.gov.justice.digital.hmpps.nomisuserrolesapi.jpa.repository.LocalAdminAuthorityRepository
+import uk.gov.justice.digital.hmpps.nomisuserrolesapi.jpa.repository.UserPersonDetailRepository
+import java.time.LocalDate
+
```

Comment:
There will be further refactoring of this when it needed for e2e tests, probably make it a Spring brean and reduce Duplication